test/tic_tac_clojure/round_test.clj

```diff
@@ -0,0 +1,26 @@
+(ns tic-tac-clojure.round-test
+  (:require [clojure.test :refer :all]
+            [tic-tac-clojure.player :refer :all]
+            [tic-tac-clojure.round :refer :all]
+            [tic-tac-clojure.sample-boards :refer :all]))
+            
+(def human-x-player
+  (Human. "X" true))
```

Comment:
Regarding the exception that you're seeing, try replacing `Human.` with `->Human` (defrecord constructor function) or `tic_tac_clojure.player.Human`.

I'm not 100% sure of the specifics, but because Clojure is compiled to Java, we need to replace hyphens with underscores for Java interoperability as hyphens are not valid in Java class names.